croud/cmd.py

```diff
@@ -290,6 +290,88 @@ def user_id_arg(
     opt_args.add_argument("--user", type=str, help="User ID.", required=False)
 
 
+def product_tier_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument("--tier", type=str, help="Product tier.", required=True)
+
+
+def product_unit_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument("--unit", type=int, help="Product unit.", required=True)
+
+
+def product_name_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument(
+        "--product-name", type=str, help="Product name.", required=True
+    )
+
+
+def crate_version_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument("--version", type=str, help="Crate version.", required=True)
+
+
+def crate_username_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument("--username", type=str, help="Crate username.", required=True)
+
+
+def crate_password_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
+    req_args.add_argument("--password", type=str, help="Crate password.", required=True)
```

Comment:
shouldn't we name it `CrateDB` ?

```suggestion
    req_args.add_argument("--password", type=str, help="CrateDB password.", required=True)
```